standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestGetTableMeta.java

```diff
@@ -233,11 +248,11 @@ public void testGetTableMeta() throws Exception {
 
     tableMetas = client.getTableMeta("testpartdb*", "*", Lists.newArrayList(
             TableType.EXTERNAL_TABLE.name()));
-    assertTableMetas(new int[]{0, 1, 3}, tableMetas);
+    assertTableMetas(new int[]{0}, tableMetas);
```

Comment:
Now that this test correctly creates MANAGED and EXTERNAL tables (because we now declare our HMS client capability for acid tables), this test output changes back to what I think was originally intended prior to  HIVE-21838.